docs/trulens/evaluation/running_feedback_functions/existing_data.md

```diff
@@ -58,7 +58,7 @@ The parameters you'll use with `VirtualRecord` are the same as those for `Record
 In the example below, we add two records. Each record includes the inputs and outputs for a context retrieval component. Remember, you only need to provide the information that you want to track or evaluate. The selectors are references to methods that can be selected for feedback, as we'll demonstrate below.
 
 ```python
-from trulens.instrument.virtual import VirtualRecord
+from trulens.instrument.core import VirtualRecord
```

Comment:
Bad replace?

Reply:
Also a few more of these.

Reply:
Ohh, nm. I see. Why not keep "trulens.instrument.virtual" ?

Reply:
I do realize if things are put into `trulens.instrument` without core, we cannot have `__init__.py` there but if we were to follow the no aliases in inits guideline, we would be ok with this.